### ISSUE TITLE
Using scan!() to read multiple values from a string source

### ISSUE THREAD

**Issue by alexmbird** [oli-obk/rust-si#8]
I'm trying to parse multiple values from a string with scan!() in Rust 1.8, but have run into trouble.  This is more likely a result of my n00b status with Rust than a problem with scan!.

Calling scan!() on a string in a single-file project works fine, just as the examples show.

However, when I move the use of scan!() out to another .rs file imported into the first, the project ceases to build.  

```
mock@MockAir12 ~/P/P/scan_test> cargo build
   Compiling scan_test v0.1.0 (file:///Users/mock/Projects/Personal/scan_test)
<text_io macros>:18:19: 18:40 error: failed to resolve. Use of undeclared type or module `std::str` [E0433]
<text_io macros>:18 } ; let s = match std:: str:: from_utf8 ( & s ) {
                                      ^~~~~~~~~~~~~~~~~~~~~
src/parser.rs:4:5: 4:42 note: in this expansion of scan! (defined in <text_io macros>)
<text_io macros>:18:19: 18:40 help: run `rustc --explain E0433` to see a detailed explanation
```

I've thrown together some example code at https://github.com/alexmbird/scan_test.

**Comment by oli-obk** [comment]
good catch

> This is more likely a result of my n00b status with Rust than a problem with scan!.

Nope, it's my inability to write proper macros ;)

**Comment by oli-obk** [comment]
published as `0.1.5`

**Comment by alexmbird** [comment]
Got it, works great.  Thanks :)
